cost-analyzer/values.yaml

```diff
@@ -584,6 +584,61 @@ kubecostModel:
   # The name of the Secret containing a bucket config for Federated storage. The contents should be stored
   # under a key named federated-store.yaml.
   # federatedStorageConfigSecret: ""
+  # federatedStorageJSON: |-
```

Comment:
Because this ends up getting templated into a YAML file `federated-store.yaml`. Would it make more sense for the user to pass in YAML here instead of JSON?

It could look something like

```yaml
kubecostModel:
  federatedStorageYaml: |-
    type: S3
    config:
      bucket: YOUR_BUCKET_NAME
      endpoint: s3.amazonaws.com
      region: YOUR_REGION
```

Reply:
@williamkubecost

Reply:
@thomasvn made a few updates!